packages/chart/src/hooks/useScales.ts

```diff
@@ -314,10 +315,13 @@ const composeYScale = ({ min, max, yMax, config, leftMax }) => {
   const scaleFunc = config.useLogScale ? scaleLog : scaleLinear
 
   if (config.visualizationType === 'Combo') max = leftMax
+
+  const yRange = config.visualizationType === 'Bump Chart' ?  [30,yMax] : [yMax,1]
```

Comment:
It looks like the original was set to [yMax, 0] here

Reply:
Changed back to the original [yMax, 0]. I'm not seeing anything visually different so not sure why I adjusted it before